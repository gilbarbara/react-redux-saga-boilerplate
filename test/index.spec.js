import React from 'react';
import { ActionTypes, XHR } from 'constants/index';
import '../app/scripts/index';

jest.mock('redux-persist/lib/integration/react', () => ({
  PersistGate: () => (<div />),
}));

describe('Constants:ActionTypes', () => {
  it('should match the snapshot', () => {
    expect(ActionTypes).toMatchSnapshot();
  });
});

describe('Constants:XHR', () => {
  it('should match the snapshot', () => {
    expect(XHR).toMatchSnapshot();
  });
});
