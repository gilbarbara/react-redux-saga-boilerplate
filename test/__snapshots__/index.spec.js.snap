// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Constants:ActionTypes should match the snapshot 1`] = `
Object {
  "HIDE_ALERT": "HIDE_ALERT",
  "SHOW_ALERT": "SHOW_ALERT",
  "USER_LOGIN_FAILURE": "USER_LOGIN_FAILURE",
  "USER_LOGIN_REQUEST": "USER_LOGIN_REQUEST",
  "USER_LOGIN_SUCCESS": "USER_LOGIN_SUCCESS",
  "USER_LOGOUT_FAILURE": "USER_LOGOUT_FAILURE",
  "USER_LOGOUT_REQUEST": "USER_LOGOUT_REQUEST",
  "USER_LOGOUT_SUCCESS": "USER_LOGOUT_SUCCESS",
}
`;

exports[`Constants:XHR should match the snapshot 1`] = `
Object {
  "FAIL": "FAIL",
  "SUCCESS": "SUCCESS",
}
`;
